### DISCUSSION TITLE
Link to waypoint (spoke) only usable once ?

### DISCUSSION THREAD

**Discussion by Letitbe133** [mozilla/hubs#2949] [General Questions & Discussion] [answered]
Hi there, just tested using a link to teleport to waypoint in the same room and works fine :thumbsup: . Pb is waypoint name stays appended to room url preventing me from using teleport link again. Am I doing something wrong here ?

**Comment by johnshaughnessy** [comment, marked as answer]
This is a known issue and should be easy to fix. Meant to get to this last week but didn't quite get to it. Will post a message here and ping you when a fix is deployed. 

More info here https://github.com/mozilla/hubs/issues/2897#issuecomment-677988368

**Reply by Letitbe133** [reply]
Thx a lot for your reply ☺👍

**Reply by johnshaughnessy** [reply]
@Letitbe133 Fixed on hubs.mozilla.com https://hubs.mozilla.com/4ukSx32/glossy-eager-terrain
Will be fixed on hubs cloud instances after the next deployment (which should happen next week)

**Reply by Letitbe133** [reply]
Cool @johnshaughnessy 👍. Sorry for late reply 🙏
